app/assets/stylesheets/active_admin/structure/_main_structure.scss

```diff
@@ -15,7 +15,7 @@
     width: 100%;
 
     #main_content{
-      margin-right: 300px;
+      margin-right: $sidebar-width + $section-padding * 2;
```

Comment:
This should be:
```suggestion
      margin-right: $sidebar-width + ($section-padding * 2);
```

Reply:
`*` should always have higher precedence than `+`, right?

Reply:
Yes but this improves readability here as its easy to miss.

Reply:
Sure!